space_packet_parser/xtce/definitions.py

```diff
@@ -74,16 +74,18 @@ def __init__(
         author : Optional[str]
             Optional author name to include in XML when serializing.
         """
-        self._parameter_type_cache = {}
-        self._parameter_cache = {}
-        self._sequence_container_cache = {}
+        self.parameter_types = {}
+        self.parameters = {}
+        self.containers = {}
```

Comment:
Is `containers` the right name for this, or do we want to follow the XTCE naming and keep it as `sequence_containers`? I'm indifferent, these are honestly just things used internally and will rarely be used by an end-user.

Reply:
I chose `containers` over `sequence_containers` because it's more concise and it contains the objects parsed out of the `<ContainerSet>` element. I just checked the element description and `SequenceContainer` is the only valid child of `ContainerSet` so I think we could use either semantic.